test/functional/ui/float_spec.lua

```diff
@@ -2041,11 +2041,11 @@ describe('float window', function()
           [2:----------------------------------------]|*6
           [3:----------------------------------------]|
         ## grid 2
-            neeed some dummy                      |
-            background text                       |
-            to show the effect                    |
-            of color blending                     |
-            of border shadow                      |
+          neeed some dummy                        |
```

Comment:
These changes are because `n.insert()` dedents, and until now, the dedent was not always correct.